src/main/java/core/basesyntax/workwithfiles/WriterImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.workwithfiles;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriterImpl implements Writer {
+    @Override
+    public void write(Map<String, Integer> mapWithResults, String newFileName) {
+        File file = new File(newFileName);

```

Comment:
I see too many strings and it should be constants and in method write use prepared result in String format